dist/docker-images/ziti-router/.gitignore

```diff
@@ -1,2 +1 @@
-.env
-compose.override.yml
```

Comment:
Stop ignoring standard compose override filename so we can use it to vend a standard set of overrides for the router tproxy mode that grant the necessary additional privileges.